docs/reference/diagnostic-tools.md

```diff
@@ -18,15 +18,15 @@ Here are several tools you can use to investigate and diagnose common issues wit
 
 ## IPFS check
 
-[IPFS Check](https://check.ipfs.network/) helps determine the retrievability of a CID from IPFS Mainnet, either from a specific peer given a multiaddress, or from multiple providers.
+[IPFS Check](https://check.ipfs.network/) helps determine the retrievability of a CID from IPFS [Mainnet](../concepts/glossary.md#mainnet), either from a specific peer given a multiaddress, or from multiple providers.
 
 @[youtube](XeNOQDOrdC0)
 
 Each error type output by the tool can indicate a solution to your problem:
 
 - **Could not connect to the multiaddr** indicates that machines on the internet cannot talk to your machine. Fix your firewall, add port forwarding, or use a relay.
 - **Could not find address in the DHT** indicates that your machine is either not connected to the Amino DHT (even as a client), or it is not advertising the address that you are using to test.
-- **Multihash not advertised in the DHT** indicates that your machine has not advertised that it has the requested content in the Amino DHT. If you're using Kubo, enable the [Accelerated DHT Client](https://github.com/ipfs/go-ipfs/blob/master/docs/experimental-features.md#accelerated-dht-client).
+- **Multihash not advertised in the DHT** indicates that your machine has not advertised that it has the requested content in the Amino DHT. 
```

Comment:
Rationale: we know there could be other reasons, such as people just unknowingly disabling announcements via `lowpower` profile (https://github.com/ipfs/kubo/pull/10524). 

Let's remove this kubo-specific suggestion, as it may do more harm than good (we know that it makes Kubo-based gateway perform worse with accelerated client enabled ([notes](https://www.notion.so/Kubo-0-29-0-vs-0-30-0-rc3-924a533a4815441983713a87f0f164e9?pvs=4#23e1c0d7c3d6442fbeb723ddb47d52c4)))